open_discussions/permissions_test.py

```diff
@@ -0,0 +1,51 @@
+"""Tests for permissions"""
+import pytest
+
+from open_discussions.permissions import (
+    JwtIsStaffPermission,
+    JwtIsStaffOrReadonlyPermission,
+)
+
+
+@pytest.mark.parametrize('perm', [
+    JwtIsStaffPermission(),
+    JwtIsStaffOrReadonlyPermission(),
+])
+def test_staff_anonymous_users_blocked(mocker, perm):
+    """
+    Test that anonymous users are not allowed
+    """
+    assert perm.has_permission(mocker.Mock(), mocker.Mock()) is False
+
+
+def test_staff_deny_non_staff_user(mocker, jwt_token):
+    """
+    Test that nonstaff users are not allowed
+    """
+    perm = JwtIsStaffPermission()
+    request = mocker.Mock(auth=jwt_token)
+    assert perm.has_permission(request, mocker.Mock()) is False
+
+
+@pytest.mark.parametrize('perm', [
+    JwtIsStaffPermission(),
+    JwtIsStaffOrReadonlyPermission(),
+])
+def test_staff_allow_staff_user(mocker, staff_jwt_token, perm):
+    """
+    Test that staff users are allowed
+    """
+    request = mocker.Mock(auth=staff_jwt_token)
+    assert perm.has_permission(request, mocker.Mock()) is True
+
+
+@pytest.mark.parametrize('method,result', [
+    ('get', False),
```

Comment:
should this be testing other verbs too?

Reply:
:man_facepalming:

Reply:
that's a sad facepalm graphic

Reply:
hahaha

Reply:
actually that tuple should be `('GET', True)` for the bonus facepalm points